C4/Budgets.pm

```diff
@@ -349,6 +350,11 @@ sub GetBudgetSpent {
     my ($shipmentcost_sum) = $sth->fetchrow_array;
     $sum += $shipmentcost_sum;
 
+    my $adjustments = Koha::InvoiceAdjustments->search({budget_id => $budget_id, closedate => { '!=' => undef } },{ join => 'invoiceid' });
```

Comment:
erm.. you're joining on invoiceid but I can't see you're using this join at all here.